woocommerce/woocommerce-admin
# Release/0.23.1

Adresses the issue raised in https://github.com/woocommerce/woocommerce-admin/issues/3394

### Test Instructions

1. Spin up a Jurassic Ninja site with WP Log Viewer and WooCommerce
2. Install the 0.23.1 plugin release https://github.com/woocommerce/woocommerce-admin/releases/tag/v0.23.1-plugin
3. Activate and connect Jetpack
4. Perform a resync here https://jetpack.com/support/debug/
5. View the logs and ensure no Onboarding errors occur. Specifically the one described in https://github.com/woocommerce/woocommerce-admin/issues/3394
